src/Sylius/Behat/Context/Ui/Admin/ManagingProductAttributesContext.php

```diff
@@ -233,6 +254,25 @@ public function iWantToSeeAllProductAttributesInStore()
     }
 
     /**
+     * @When /^(the administrator) changes the value "([^"]*)" to "([^"]*)" of (this product attribute)$/
```

Comment:
It doesn't feel right reading this step, what about the something like: the administrator changes this product attribute value "foo" to "bar"?